modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C

```diff
@@ -32,8 +36,20 @@ ComputeLinearElasticPFFractureStress::ComputeLinearElasticPFFractureStress(
         getParam<MaterialPropertyName>("F_name"), getVar("c", 0)->name(), getVar("c", 0)->name())),
     _d2Fdcdstrain(declareProperty<RankTwoTensor>("d2Fdcdstrain")),
     _dstress_dc(
-        declarePropertyDerivative<RankTwoTensor>(_base_name + "stress", getVar("c", 0)->name()))
+        declarePropertyDerivative<RankTwoTensor>(_base_name + "stress", getVar("c", 0)->name())),
+
+    _H0_pos(declareProperty<Real>("H0_pos")),
+    _H0_pos_old(getPropertyOld<Real>("H0_pos"))
+
+{
+}
+
+void
+ComputeLinearElasticPFFractureStress::initQpStatefulProperties()
 {
+  ComputeStressBase::initQpStatefulProperties();
+  _H0_pos[_qp] = 0.0;
+  _H0_pos_old[_qp] = 0.0;
```

Comment:
You only need to initialize the _current_ property value here, not the old properties!